pythonforandroid/toolchain.py

```diff
@@ -376,6 +376,11 @@ def __init__(self):
             dest='local_recipes', default='./p4a-recipes',
             help='Directory to look for local recipes')
 
+        generic_parser.add_argument(
+            '--activity-class-name',
+            dest='activity_class_name', default='org.kivy.android.PythonActivity',
```

Comment:
I wish we had a single source of truth shared with https://github.com/kivy/python-for-android/pull/2248/files#diff-c439e980dc587602fda87459815f09feR450 for the `org.kivy.android.PythonActivity` value, but I can live without

Reply:
@AndreMiras  you mean that there are two settings now that are storing same value? : `android.entrypoint` and `android.activity_class_name` ?